plugins/outputs/plugin.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
+ * you may not use this file except in compliance with the License.
+ * Full License text is in the LICENSE file included in the root of this repository.
+ */
+
+package main
```

Comment:
I'd add some high-level descriptions for this plugin here.

```suggestion
// The output plugin ...

package main
```